### PR TITLE
MediaStreamTrack.stop(): Address early return for non-locally sourced tracks (issue #314)

### PR DESCRIPTION
**Pull request by adam-be**

### CODE REVIEW

File: getusermedia.html [hunk 1]
```diff
@@ -545,7 +545,11 @@
     <section>
       <h2>MediaStreamTrack</h2>
       <p>A <code><a>MediaStreamTrack</a></code> object represents a media
-      source in the User Agent. Several <code><a>MediaStreamTrack</a></code>
+      source in the User Agent. A source, unless stated otherwise, is a
+      <dfn>local source</dfn> representing a device connected to the User
+      Agent. Other specifications may define sources for <code>
```
Comment (alvestrand) [inline getusermedia.html]: Suggest "One example of a source is a device connected to the User Agent".

**Comment by alvestrand** [comment]
Hotlink #314

**Comment by adam-be** [comment]
This PR is related to #520 in webrtc-pc

**Comment by adam-be** [comment]
@burnburn, please have a look at the updated version.

**Comment by burnburn** [comment]
LGTM

**Pull request merged**